rust/catalyst-types/src/hashes.rs

```diff
@@ -165,6 +169,29 @@ impl<'a, C, const BYTES: usize> minicbor::Decode<'a, C> for Blake2bHash<BYTES> {
     }
 }
 
+impl<const BYTES: usize> SerializeValue for Blake2bHash<BYTES> {
```

Comment:
I would rather not doing that, because maintaining and interacting with the db is not responsibility of this crate.